### PR TITLE
Rework the Seed cluster backups section

### LINKED ISSUE #746: Simplify "Cluster backups" section
https://docs.kubermatic.com/kubermatic/master/guides/installation/add_seed_cluster_ce/

When adding a seed cluster, a StorageClass for `kubermatic-storage` is required. The section describing this part of the setup is really hard to follow. My suggestion would be to change it as follows:

1. Inform the user that a storage class of a specific name is required, define if there are any specific requirements for it (HDD/SSD/object, etc.)
2. Suggest possible solutions, e.g. minio, EBS on AWS, similar on GCP...
3. Make (each of?) possible solutions a separate article, link from the main document.
4. How does the s3-exporter connect to the backups if the customer is not using minio, just ebs? I think this belongs in the part describing setup with minio.

### PR DESCRIPTION
**Pull request by wurbanski**
I tried to convey more information about:
* how is Minio used,
* what exactly is being configured
* how to change the configuration

Closes #746

### CODE REVIEW

File: content/kubermatic/master/guides/installation/add_seed_cluster_CE/_index.en.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ about the cluster relationships.
 Compared to master clusters, seed clusters are still mostly manually installed. Future versions of KKP
 will improve the setup experience further.
 
-When using Helm 2, install Tiller into the seed cluster first:
+If you're using Helm 2, install Tiller into the seed cluster first:
```
Comment (shaase-ctrl) [inline content/kubermatic/master/guides/installation/add_seed_cluster_CE/_index.en.md:23]: I do not know if helm 2 is really an option here?

**Review by kron4eg — APPROVED** [review]
/approve

File: content/kubermatic/master/guides/installation/add_seed_cluster_CE/_index.en.md (hunk 1 above)
Comment (kron4eg) [reply content/kubermatic/master/guides/installation/add_seed_cluster_CE/_index.en.md:23]: Unfortunately it is
